client/view.js

```diff
@@ -3,29 +3,35 @@ export default class View
 	#bus = null;
 	#playerRows = {};
 	#playerId = null;
+	#playerName = null;
 
 	constructor($container, bus, playerId)
 	{
 		this.#bus = bus;
 		this.#initBus();
-
 		this.#playerId = playerId;
 
 		$container.querySelector('button.js-create-game').addEventListener('click', e => {
-			this.#bus.publish('create-game');
+			e.preventDefault();
+			this.#bus.publish('create-game',this.playerName.value);
 		}, false);
 
 		$container.querySelector('button.js-join-game').addEventListener('click', e => {
 			e.preventDefault();
-			this.#bus.publish('join-game', $container.querySelector('input[name=game-id]').value);
+			this.#bus.publish('join-game', 

```

Comment:
can we make this all on one line please, line wrapping should be an IDE setting (i.e. virtual) - not something imposed by line breaks.